tests/test_wave_01.py

```diff
@@ -145,8 +145,8 @@ def test_moves_movie_from_watchlist_to_watched():
     # *******************************************************************************************
     # ****** Add assertions here to test that the correct movie was added to "watched" **********
     # *******************************************************************************************
+    assert updated_data["watched"][1] is HORROR_1
```

Comment:
In this case I suggest `assert HORROR_1 in updated_data["watched"]`. The requirements don't specify in what order the the movie should be added to the "watched" list.